docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,839 @@
+
+> danger
+> Creating a command with the same name as an existing command for your application will overwrite the old command.
+
+Create a new guild command. New guild commands will be available in the guild immediately. Returns `200` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand) object.
+
+###### JSON Params
+
+| Field       | Type                                                                                              | Description                    |
+|-------------|---------------------------------------------------------------------------------------------------|--------------------------------|
+| name        | string                                                                                            | 3-32 character command name    |
+| description | string                                                                                            | 1-100 character description    |
+| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandoption) | the parameters for the command |
+
+## Edit Guild Application Command % PATCH /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands/{command.id#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand}
+
+Edit a guild command. Updates for guild commands will be available immediately. Returns `200` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand) object.
+
+###### JSON Params
+
+| Field       | Type                                                                                              | Description                    |
+|-------------|---------------------------------------------------------------------------------------------------|--------------------------------|
+| name        | string                                                                                            | 3-32 character command name    |
+| description | string                                                                                            | 1-100 character description    |
+| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandoption) | the parameters for the command |
+
+
+## Delete Guild Application Command % DELETE /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/commands/{command.id#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand}
+
+Delete a guild command. Returns `204` on success.
+
+## Create Interaction Response % POST /interactions/{interaction.id#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction}/{interaction.token#DOCS_INTERACTIONS_SLASH_COMMANDS/interaction}/callback
+
+Create a response to an Interaction from the gateway. Functions the same as [Execute Webhook](#DOCS_RESOURCES_WEBHOOK/execute-webhook)
```

Comment:
this does **not** function the same as execute webhook, should link to the Interaction Response section at the bottom